core/templates/components/question-directives/question-misconception-editor/question-misconception-editor.component.ts

```diff
@@ -25,7 +25,7 @@ import { Misconception, MisconceptionSkillMap, TaggedMisconception } from 'domai
 import { ExternalSaveService } from 'services/external-save.service';
 import { TagMisconceptionModalComponent } from './tag-misconception-modal-component';
 import { SubtitledHtmlBackendDict } from 'domain/exploration/subtitled-html.model';
-import { Rule } from 'domain/exploration/RuleObjectFactory';
+import { Rule } from 'domain/exploration/rule-object.model';
```

Comment:
Here and elsewhere, let's drop "object" and just use "rule model"? Seems redundant to say "object model" IMO